### ISSUE TITLE
Design, FE | Profile | Screenreader Accessibility Issue | Address Validation

### ISSUE THREAD

**Issue by ericasauv09** [department-of-veterans-affairs/va.gov-team#47272]
## Background

During the Accessibility Audit address validation was noted as being difficult to navigate for screenreader users. This concern was raised during the BAI pilot with Angela. We do not have a solution for this currently. 

Note: this may need to be broken up into Design and FE tickets separately. 

## Tasks
- [ ] Design to conduct research on possible solutions
- [ ] Share solutions with team, identify best option
- [ ] FE to implement solution
- [ ] Conduct Unit tests 
- [ ] Run E2E tests covering current code and regression
- [ ] All axe checks pass
- [ ] Deploy to Staging behind feature flag

## Acceptance Criteria
- [ ] Complete all unit testing
- [ ] End-to-end tests showing 0 violations
- [ ] Notify Product Manager that the change is ready to be validated on Staging
- [ ] Incorporate any changes resulting from Staging review & re-test & have re-validated
- [ ] Deploy changes to Production using feature flag

## Validation
- Who can validate this ticket? (FE, BE, Design, PM)?
- How can this work be validated?
- What updates need to be made (e.g. product outline, use cases, contact center guide)

**Comment by SKasimow** [comment]
Pending research findings...

**Comment by SKasimow** [comment]
Closing....replaced by #48537
